File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        nmap = {}
        for n in nums:
            if n not in nmap:
                nmap[n] = 1
            else:
                nmap[n]+=1

        return sorted(nmap, key=nmap.get)[-k:]

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int r = 1;
        for (int k = 1; k< nums.length; k++){
            if (nums[k] != nums[r-1]){
                nums[r] = nums[k];
                r++;
            }
        }
        return r;
    }
}